ts/deckoptions/deckoptions-base.scss

```diff
@@ -28,16 +28,18 @@
 }
 
 .tooltip-inner {
-    max-width: 300px;
     text-align: left;
+    max-width: 300px;
 
-    p {
-        margin: 0.8em 0.4em 0.8em 0.8em;
+    // marked transpiles tooltips into multiple paragraphs
+    // where trailing <p>s cause a bottom margin
+    > p:last-child {
+        display: inline;
```

Comment:
That's better, but the tooltips as a whole still need some padding in my opinion.
Looks a bit packed currently:

![Screenshot from 2021-06-12 10-21-29](https://user-images.githubusercontent.com/62722460/121770181-230ca280-cb68-11eb-8078-364d035bde68.png)

Reply:
There's also another issue with no padding applied I noticed:

![image](https://user-images.githubusercontent.com/7188844/121770279-b8a83200-cb68-11eb-957e-a96c834c49ec.png)